conformance/packages/dns-test/src/client.rs

```diff
@@ -79,13 +80,27 @@ impl Client {
     }
 }
 
-#[derive(Clone, Copy, Default)]
+#[derive(Clone, Copy)]
 pub struct DigSettings {
     adflag: bool,
     cdflag: bool,
     dnssec: bool,
     recurse: bool,
     timeout: Option<u8>,
+    edns: Option<u8>,
```

Comment:
If we're going to default to `Some(0)`, why does this need to be `Option`?

Reply:
The semantics of this field are that `Some(0)` means to use EDNS(0), by providing the `+edns=0` flag, while `None` means to not use EDNS at all, by providing the `+noedns` flag. `dig`'s default setting is `+edns=0`. I'll add a doc comment explaining this.